packages/web-frontend/src/containers/OverlayDiv/components/NotFoundPage.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+import { PrimaryButton } from '../../../components/Buttons';
+import { goHome } from '../../../actions';
+import { connect } from 'react-redux';
+
+const Container = styled.div`
+  height: 500px;
+  width: 800px;
+  max-width: 100%;
+  max-height: 100%;
+  text-align: center;
+  padding-top: 20%;
+  padding-bottom: 30px;
+`;
+
+const Image = styled.img`
+  width: 300px;
+  max-width: 100%;
+  margin-bottom: 30px;
+`;
+
+const Text = styled(Typography)`
+  font-size: 18px;
+  line-height: 21px;
+  color: white;
```

Comment:
What are your thoughts on using a theme color for text color, on the off chance that the theme color changes (e.g. `palette.text.secondary` etc etc). It's not a random hex code so should be pretty straightforward.

Reply:
Yes I agree - good suggesion